README.md

```diff
@@ -39,8 +46,19 @@ yarn               # or yarn install
 
 ### Alterar conteúdos
 
-Os ficheiros `.md` (markdown) estão na pasta `src`.
-Para alterar os links na sidebar, é necessário alterar o ficheiro `src/.vuepress/config.js`.
+Os ficheiros `.md` (markdown) estão na pasta da respetiva UC dentro de `content`.
+Assim que um ficheiro `.md` é adicionado à respetiva pasta da UC, é acessível a partir da sidebar.  
+Os ficheiros podem ainda ter diferentes categorias, aparecendo em diferentes sub-secções da sidebar consoante a mesma. Em princípio, ser-vos-á útil apenas a categoria `content`, que deve ser adicionada ao _header_ de cada ficheiro.
+
+Cada ficheiro deve conter um _header_ com a meta-informação útil correspondente a cada ficheiro, principalmente:
+
+- `title: <título>`, onde `<título>` será o título que aparece associado à página correspondente ao ficheiro nos resumos;
+
+- `description: <bullet points>`, secção que deverá indicar sucintamente os pontos importantes abordados neste capítulo dos resumos (e que aparece no _embed_ do URL ao partilhar o link da página);
+
+- `path: /<UC>/<nome-da-pagina>`, _self-explanatory_;
+
+- `type: <categoria>`, tal como mencionado acima provavelmente apenas vos interessará a categoria `content`.
```

Comment:
Maybe it'd be nice to put here a full list of available types (or maybe just in the in-depth docs in the wiki)

Reply:
I'd argue it'd be fine for it to be fully documented in the Wiki (and for us to include only the essentials here), but it's your call